ethcore/src/engines/clique/mod.rs

```diff
@@ -138,14 +144,58 @@ impl Engine<EthereumMachine> for Clique {
 
 	// called only when sealing ?
 	fn populate_from_parent(&self, header: &mut Header, parent: &Header) {
```

Comment:
yes,  it does.   enact_verified()  is the only place that uses enact() that uses  OpenBlock::new() underneath to call here, it's unnecessary and it's very confusing, that why I say we should refactor the code to seperate import/sealing process first. Otherwise there are/will be countless weird bug before and after.

Reply:
I agree that we need to have separate import/sealing paths.  I think I know an easy way to do this without modifying too much of the core block processing code.